pkg/controllers/management/auth/project_cluster/cluster_handler.go

```diff
@@ -331,12 +336,15 @@ func (l *clusterLifecycle) addRTAnnotation(obj runtime.Object, context string) (
 }
 
 func (l *clusterLifecycle) reconcileClusterCreatorRTB(obj runtime.Object) (runtime.Object, error) {
-	return apisv3.CreatorMadeOwner.DoUntilTrue(obj, func() (runtime.Object, error) {
-		cluster, ok := obj.(*apisv3.Cluster)
-		if !ok {
-			return obj, fmt.Errorf("expected cluster, got %T", obj)
-		}
+	cluster, ok := obj.(*apisv3.Cluster)
+	if !ok {
+		return obj, fmt.Errorf("expected cluster, got %T", obj)
+	}
 
+	if _, ok := cluster.Annotations[NoCreatorRBACAnnotation]; ok {
+		return obj, nil
```

Comment:
Just a question, not even a nit really.  "No" is a fine answer:  Is there any chance that a log entry of some sort might be useful here and in the project handler when we're intentionally skipping RBAC?

Reply:
I like the idea too.

Reply:
Good call! I added a debug statement here and in the project handler

Reply:
Is there any chance we can make it `Info` level instead? I wouldn't insist but I think it's very useful to leave a trace in the log under normal circumstances.

Reply:
Done